packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -18,7 +18,7 @@ const style = `
 body {
     background: linear-gradient(-45deg, #e73c7e, #23a6d5, #ee7752);
     background-size: 400% 400%;
-    animation: gradient 10s ease infinite;
+    animation: gradient 10s ease 5;
```

Comment:
🔫

Reply:
Ahh yeah.. forgot to remove that. But we should change that in another PR.. really zaps the cpu